### ISSUE TITLE
Multiple links on Pages page clip text when resized

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6789]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Bold` dropdown box, formatting links and `Powered by Tiny` links on the Pages page clips text when resized and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin user, go to https://dev.nurax.samvera.org/pages/edit?locale=en and run the Site Improve browser extension

![pages_text_clip](https://github.com/samvera/hyrax/assets/32644254/564ac694-eab3-4c80-8ee6-fe611df8097c)

**Comment by aahurford** [comment]
tested and confirmed on pg

**Comment by jlhardes** [comment]
pg.nurax has this change applied but dev.nurax does not have this update yet.

**Comment by jlhardes** [comment]
There is a difference between browser/OS combos using SiteImprove. Chrome on Mac is not catching any resize errors with this change but Chrome on Windows is still registering a resize error related to the drop-down menu in the editor. Re-opening to try some more overflow style overrides and see if that addresses this issue for SiteImprove in Chrome on Windows.

**Comment by christhepianist** [comment]
I am still seeing the same error. This time I used Firefox and SiteImprove with the same results. Here I've included the checker's help on how to fix it. I don't know if it will be helpful or not.
![Image](https://github.com/user-attachments/assets/50beed05-3a03-46e5-acb6-fdc408f73537)

On the other hand, I used the SilkTide accessibility checker and it doesn't see this issue at all.

**Comment by christhepianist** [comment]
The changes propagated to pg.nurax. Confirmed that this is no longer an issue!!
